### DISCUSSION TITLE
Memory / Ram Usage

### DISCUSSION THREAD

**Discussion by jeremiahsherrill** [spatie/laravel-health#127] [Ideas]
Unless I am missing something, I am not seeing memory usage as one of the options.

**Comment by soundboxfm** [comment]
i don't have the same problem.
I had RAM 14 GB after a short time this is fully exhausted. Have increased to 16 GB, and again after a few hours it is permanently exhausted.
I'm now at 18 GB and still the same problem again.
What can I do?
